starter-code/src/components/CountryInfo.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class CountryInfo extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      country: this.props.location.state.listOfCountries
```

Comment:
Not needed to assign the props to the state of this component. 

Props can be accessed in the `return()` without having to store them in the state first.